docs/source/admin/traffic_ops/using.rst

```diff
@@ -850,6 +867,62 @@ The CZF is an input to the Traffic Control CDN, and as such does not get generat
 
 The script that generates the CZF file is not part of Traffic Control, since it is different for each situation.
 
+.. note:: The ``"coordinates"`` section is optional and may be used by Traffic Router for localization in the case of a CZF "hit" where the zone name does not map to a Cache Group name in Traffic Ops (i.e. Traffic Router will route to the closest Cache Group(s) geographically).
+
+.. _rl-deep-czf:
+
+The Deep Coverage Zone File
++++++++++++++++++++++++++++
+The Deep Coverage Zone File (DCZF) format is similar to the CZF format but adds a ``caches`` list under each ``coverageZone``: ::
+
+  {
+    "coverageZones": {
+      "location-01": {
```

Comment:
This must not share a name with a cache group, right?

Reply:
No, it doesn't matter if they share a name because it's arbitrary, unlike in the regular CZF where they map to a cache group.